app/javascript/packs/components/RestaurantItem.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+function Rating(props) {
+  let stars = [];
+  for (let i = 0; i < props.stars; i = i + 1) {
```

Comment:
you should do this in the 'return' section:
{[1,2,3].map((star) =>
    (<div key={star} ...